### PR TITLE
Add Tasks 209-215

### PR DESCRIPTION
**Pull request by zaetsa12**

### CODE REVIEW

**Review by SvitlanaLN — COMMENTED** [review]
image
         Console.WriteLine($"Factorial of {yourValue} is {i}");

**Review by SvitlanaLN — COMMENTED** [review]
using System.Linq; 

This row is redundant.

**Review by SvitlanaLN — COMMENTED** [review]
Good job!

File: Projects/Team1/AnnaZaiets/Task3/Task3/Task213/Program.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        static void Main(string[] args)
+        {
+            //5. Користувач вводить з клавіатури число, необхідно перевернути його (число) і вивести на екран.
+            Console.WriteLine("Enter you number ");
+            int number = int.Parse(Console.ReadLine());
+            int result = 0;
+            while (number > 0)
+            {
+                if(number < 0)
```
Comment (romanukhaav) [inline Projects/Team1/AnnaZaiets/Task3/Task3/Task213/Program.cs:19]: do you need this checking inside of loop?

File: Projects/Team1/AnnaZaiets/Task3/Task3/Ефіл212/Program.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            if(number < 0)
+            {
+                number = -number;
+            }
+            for(int i=0; i<=number;i++)
+            {
+                number /= 10;
+                count++;
+            }
+            Console.WriteLine("There are " + count + " numbers in your number.");
```
Comment (romanukhaav) [inline Projects/Team1/AnnaZaiets/Task3/Task3/Ефіл212/Program.cs:26]: string interpolation is better

**Review by romanukhaav — APPROVED** [review]
great!

**Review by FeloniukM — APPROVED** [review]

**Review by Sasha380 — APPROVED** [review]

File: Projects/Team1/AnnaZaiets/Task3/Task3/Task213/Program.cs (hunk 1 above)
Comment (zaetsa12) [reply Projects/Team1/AnnaZaiets/Task3/Task3/Task213/Program.cs:19]: of course not, thank you

**Pull request merged**